src/TrackGenerator.cpp

```diff
@@ -1918,14 +2024,29 @@ void TrackGenerator::initializeSegments() {
   FSRs_to_keys = _geometry->getFSRsToKeys();
 
   /* Iterate over all Track segments and assign them each a Material */
-  for (int i=0; i < _num_azim; i++) {
-    for (int j=0; j < _num_tracks[i]; j++) {
-      for (int s=0; s < _tracks[i][j].getNumSegments(); s++) {
-        curr_segment = _tracks[i][j].getSegment(s);
-	region_id = curr_segment->_region_id;
-        Material* mat = _geometry->findFSRMaterial(region_id);
-	curr_segment->_material = _geometry->findFSRMaterial(region_id);
-	FSR_keys_map->at(FSRs_to_keys->at(region_id))->_mat_id = mat->getId();
+#pragma omp parallel
```

Comment:
I'll add tests for fsr volume correction and segment splitting here. Totally agree - track loading should be addressed in some tests, especially as we consider the considerable changes to the track objects for 3D MOC that we'll want to merge in. Best to have the tests spot potential sync issues before they get integrated.

Thanks for getting on the multi-sim tests so quickly!